src/gracenotegroup.ts

```diff
@@ -25,6 +25,10 @@ function L(...args: any) {
   if (GraceNoteGroup.DEBUG) log('Vex.Flow.GraceNoteGroup', args);
 }
 
+// Left padding to avoid that the group collides with the previous note.
+// The value increases the width and the x offset.
+const LEFT_PADDING = 2;
```

Comment:
You can use StaveNote.minNoteheadPadding.  It is actually defined in the font metric.

Reply:
Done, thank you!